app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:id])
+    @posts = @user.posts
+  end
+
+  def show
+    @posts = @user.posts.find(params[:id])
+  end
+end
```

Comment:
- [ ] There should be a corresponding view file for each controller, kindly add the files to your view folder